packages/form/src/LionForm.js

```diff
@@ -8,43 +7,53 @@ import { LionFieldset } from '@lion/fieldset';
  * @extends LionFieldset
  */
 // eslint-disable-next-line no-unused-vars
-export class LionForm extends DelegateMixin(LionFieldset) {
-  get delegations() {
-    return {
-      ...super.delegations,
-      target: () => this.formElement,
-      events: [...super.delegations.events, 'submit', 'reset'],
-      methods: [...super.delegations.methods, 'submit', 'reset'],
-    };
-  }
-
+export class LionForm extends LionFieldset {
   constructor() {
     super();
+    this.__boundFormElementEvents = this._formElementEvents.bind(this);
     this.__boundSubmit = this._submit.bind(this);
     this.__boundReset = this._reset.bind(this);
   }
 
   connectedCallback() {
     super.connectedCallback();
+    this.formElement.addEventListener('submit', this.__boundFormElementEvents);
     this.addEventListener('submit', this.__boundSubmit);
+    this.formElement.addEventListener('reset', this.__boundFormElementEvents);
     this.addEventListener('reset', this.__boundReset);
   }
 
   disconnectedCallback() {
     super.disconnectedCallback();
+    this.formElement.removeEventListener('submit', this.__boundFormElementEvents);
     this.removeEventListener('submit', this.__boundSubmit);
+    this.formElement.removeEventListener('reset', this.__boundFormElementEvents);
     this.removeEventListener('reset', this.__boundReset);
   }
 
   get formElement() {
     return this.querySelector('form');
   }
 
+  submit() {
+    this.formElement.submit();
+  }
+
+  reset() {
+    this.formElement.reset();
+  }
+
   /**
    * As we use a native form there is no need for a role
    */
   _setRole() {} // eslint-disable-line class-methods-use-this
 
+  _formElementEvents(ev) {
```

Comment:
just a suggestion maybe do it in a similar manner like
https://github.com/ing-bank/lion/pull/228/commits/97b958fffe47e144c5d8e3d129bf5518f9d82a1c#diff-0ecc97735e01830c554cf791bf2f1ebaR76
?